src/v2i-hub/PortDrayagePlugin/src/PortDrayagePlugin.cpp

```diff
@@ -30,17 +31,37 @@ PortDrayagePlugin::~PortDrayagePlugin() {
 }
 
 void PortDrayagePlugin::UpdateConfigSettings() {
-	// Update configuration
+
 	lock_guard<mutex> lock(_cfgLock);
+
+	// Database configuration
 	GetConfigValue<string>("Database_Username", _database_username);
 	GetConfigValue<string>("Database_Password", _database_password);
 	GetConfigValue<string>("Database_IP",_database_ip);
 	GetConfigValue<uint16_t>("Database_Port",_database_port);
 	GetConfigValue<string>("Database_Name", _database_name);
-	std::string loglevel;
-	GetConfigValue<string>("LogLevel", loglevel);
 
-	FILELog::ReportingLevel() = FILELog::FromString(loglevel);
+	// Port Drayage Web Service Configuration
+	uint16_t polling_frequency;
+	uint16_t polling_timeout;
+	std::string host;
+	uint16_t port;
+	bool secure;
+	// Host address 
+	GetConfigValue<string>("Webservice_Host", host);
+	// Host Port
+	GetConfigValue<uint16_t>("Webservice_Port", port);
+	// True for HTTPS
+	GetConfigValue<bool>("Webservice_Secure", secure);
+	// Polling Frequency in seconds
+	GetConfigValue<uint16_t>("Webservice_Polling_Frequency", polling_frequency);
+
+	client = new WebServiceClient(host, port, secure, polling_frequency);
```

Comment:
delete the pointer after finish using it. The delete might be in the destructor if this client is a global variable.